compiler/x/codegen/OMRX86Instruction.cpp

```diff
@@ -4370,12 +4331,6 @@ generateLabelInstruction(TR_X86OpCodes     opCode,
    return instr;
    }
 
-TR::X86RestoreVMThreadInstruction  *
-generateRestoreVMThreadInstruction(TR::Node          *node,
-                                   TR::CodeGenerator *cg)
-   {
-   return new (cg->trHeapMemory()) TR::X86RestoreVMThreadInstruction(RestoreVMThread, node, cg);
-   }
 
```

Comment:
Can we remove the additional new lines? Similarly in a few other places in this PR.